src/test/java/org/gridsuite/actions/test/GenerateScriptFromFiltersTest.java

```diff
@@ -45,13 +45,13 @@ public void generateScriptTestInjection() {
             "        contingency(equipment.id) { equipments equipment.id }\n" +
             "  }\n" +
             "}\n", formToScript.generateGroovyScriptFromForm(new FormContingencyList(
-            null,
+            null, null,
```

Comment:
To avoid those form of constructorswith some null values, we could add a 2 parameters less constructor which set id and date to null ?

Reply:
done for ScriptContingencyList and FormContingencyList